proofs/alf/cvc5/programs/Strings.smt3

```diff
@@ -138,11 +138,16 @@
   (str.substr s 0 n)
 )
 
-; Get the term corresponding to the suffix of term s of fixed length n.
+; Get the term corresponding to the suffix of term s starting from position n.
 (define skolem_suffix_rem ((U Type :implicit) (s (Seq U)) (n Int))
   (str.substr s n (- (str.len s) n))
 )
 
+; Get the term corresponding to the suffix of term s of length n.
+(define $str_suffix_len ((U Type :implicit) (s (Seq U)) (n Int))
+  (str.substr s (- (str.len s) n) n)
+)
+
```

Comment:
We are in the process of updating to `$str_`, once all open PRs are merged then at some point I will do a global replace. This is mostly just to avoid git clashes.